src/promptflow-evals/promptflow/evals/evaluators/similarity/flow/validate_inputs.py

```diff
@@ -0,0 +1,11 @@
+from promptflow import tool

```

Comment:
please use promptflow.core import tool

Reply:
Working on a follow up PR to resolve this comment.